Source/Core/Core/ConfigLoaders/MovieConfigLoader.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/CommonPaths.h"
```

Comment:
`<cstring>`, `<string>`, `"Common/CommonFuncs.h"`, and `"Common/CommonTypes.h"` need to be included.